app/router.js

```diff
@@ -14,6 +14,7 @@ Router.map(function() {
     });
   });
   this.route('exit');
+  this.route('complete');
```

Comment:
Can you please clarify why the router defines the `complete` route twice?

(if this is legacy of a previous change, please glance over the PR to make sure nothing else is left over)